### ISSUE TITLE
Copper Beaten Skull

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7481]
**Preferred term label:**
Copper Beaten Skull

**Synonyms**

**Definition (free text, please give PubMed ID)**
Diffuse prominent gyral impressions on the inner table of skull vault said to resemble beaten copper.
PMID: 31466955

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 619451 
PMID: 33991472  8/8

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
